src/gui/WelcomeWidget.ui

```diff
@@ -128,6 +128,13 @@
     </widget>
    </item>
    <item>
+    <widget class="QPushButton" name="buttonImportCSV">
+     <property name="text">
+      <string>Import from plain text CSV file</string>
```

Comment:
Simply `Import from CSV` would be more consistent with the other import option `Import from KeePass 1`.